### PR TITLE
FIX: fixing docs errors

### PR DESCRIPTION
**Pull request by PatrickSampaioUSP**

### CODE REVIEW

**Comment by Gui-FernandesBR** [comment]
@PatrickSampaioUSP tkns for helping,

We need address pytests failure before merging, it seems there's a problem on `ipywidgets` imports.

Also, shouldn't be better merging into develop branch before master so we guarantee they before stay syncronized ?

**Comment by PatrickSampaioUSP** [comment]
Usually we should merge hot_fixes directly to the master, and then after we update the develop branch

**Review by Gui-FernandesBR — APPROVED** [review]
well done!

**Review by giovaniceotto — APPROVED** [review]
Since all relevant changes are done in the docs folder, I am confortable merging it directly to the master branch to fix such docs issue.

Furthermore, the changes made outside the docs folder are small and simple to review.

I am approving this PR and merging it.

**Comment by Gui-FernandesBR** [comment]
@PatrickSampaioUSP @giovaniceotto need some extra help here,

apperantly the documentation is still failling, do you know how to solve this ? 

image

File: rocketpy/__init__.py [hunk 1]
```diff
@@ -22,6 +22,7 @@
 __status__ = "Production"
 
 from .Environment import Environment
+from .EnvironmentAnalysis import EnvironmentAnalysis
```
[the remark this answers is not in the document]
Comment (PatrickSampaioUSP) [reply rocketpy/__init__.py:25]: Yes, that would be a problem?

**Comment by PatrickSampaioUSP** [comment]
Screenshot from 2022-07-19 07-28-53

**Comment by PatrickSampaioUSP** [comment]
Yeah, by some bizarre reason he is not finding the timezonefinder>=6.0.2

**Comment by Gui-FernandesBR** [comment]
> Yeah, by some bizarre reason he is not finding the timezonefinder>=6.0.2

Ok so how can we fix that? Unfortunately there's no possiblity of re-opening te pull request, but as long we don't have restored documentation page completely, our job is not done.

In the screen shot you presented above, it seems timezonefinder requires a specifically different python version. Since we only use this on the Environment Analysis file, shall we remove timezone from requirements and stop using it from rocketpy package until we discover other solution for the `timezonefinder` ?

**Pull request merged**